src/components/grid/index.js

```diff
@@ -3,36 +3,44 @@ import React from 'react';
 import classNames from 'classnames';
 import styles from './style.css';
 
-const gridClassNames = classNames(
-  styles.grid,
-  styles.withGutter,
-  styles.fit
-);
-
-export class Grid extends React.Component {
-  render () {
-    return (
-      <div className={gridClassNames}>
-        {this.props.children}
-      </div>
-    );
+export function Grid (props) {
+  let fitClass;
+  let gridClassNames;
+  if (props.fit) {
+    fitClass = styles.fit;
   }
+  gridClassNames = classNames(
+   styles.grid,
+   styles.withGutter,
+   fitClass
```

Comment:
You could say `{[styles.fit]: props.fit}` here and save yourself the `if` above.